vendor/engines/ldap_authentication/ldap_authentication.gemspec

```diff
@@ -16,6 +16,6 @@ Gem::Specification.new do |s|
   s.files = Dir["{app,config,db,lib}/**/*", "README.md"]
   s.test_files = Dir["spec/**/*"]
 
-  s.add_dependency "rails", "~> 4.2.7.1"
+  s.add_dependency "rails", "~> 4.2.9"
```

Comment:
My guess is that it's part of the default generated file. I think it's probably safe to get rid of, but we can do that down the line.